sdk-libs/compressible-client/decompress-mint.md

```
@@ -0,0 +1,514 @@
+```rust
+pub mod decompress_atas;
+pub mod decompress_mint;
+pub mod get_compressible_account;
+
+pub use decompress_atas::*;
+pub use decompress_mint::*;
+```
+
+## Flow Diagram
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)